### DISCUSSION TITLE
GitHub

### DISCUSSION THREAD

**Discussion by Sheriffmustapha** [community/community#143762] [GitHub Education]
Hi everyone,
Hi everyone,

I'm a student interested in applying for GitHub's Student Developer Pack to access resources and tools that would really help with my studies. However, I want to make sure I submit the correct information and qualify without any issues.

Could someone please clarify the eligibility requirements? Also, what type of documentation do I need to provide to ensure my application is accepted? Any tips or advice would be greatly appreciated!

Thanks in advance!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Sheriffmustapha ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and including more details. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
